index.html

```diff
@@ -4545,28 +4545,172 @@ <h2>
       </p>
     </section>
 
-    <section>
-      <h2>
-Key Revocation and Recovery
-      </h2>
 
-      <p>
+
+<section>
+  <h2>
+  Verification Method Rotation
+  </h2>
+  
+  <p>
+    Verification method rotation is a proactive security measure.
+  </p>
+  
+  <p>
+    When a verification method has been active for a long time, 
+    or used for many operations, a controller might wish to perform a rotation.
+  </p>
+
+  <p class="advisement">
+    It is considered a best practice to perform verification method rotation on a regular basis.
+  </p>
+  
+  <p>
+    Section <a href="#method-operations"></a> specifies the <a>DID</a> operations to
+    be supported by a <a>DID method</a> specification, 
+    including <a href="#update">update</a>
+    which MUST be used to perform a verification method rotation.
+  </p>
+  
+  <p>
+    A controller performs a rotation when they change 
+    <code>publicKeyJwk</code> or <code>publicKeyBase58</code>, or other verification material
+    without changing the <code>id</code> of the verification method.
+  </p>
+  
+  <p>
+    Unlike revocation, rotation allows for new verifications associated with the <code>id</code>, 
+    to be valid, while invalidating old verifications that were produced with the older material.
```

Comment:
I don't agree that key rotation invalidates old verifications. That is not the case with passwords or SSH keys; why should we claim it is true of DID keys? You are wanting to use verification methods to produce ephemeral tokens and are claiming that is a general property of the entire DID ecosystem. I disagree.

Reply:
removing an ssh key or a jwk from the associated collection does cause a verify to reject them... perhaps you can propose some alternate language that applies equally to SSH / GPG / OIDC?

GitHub will not accept commits from "rotated" ssh keys.